app/controllers/membership_applications_controller.rb

```diff
@@ -68,6 +68,11 @@ def information
 
   end
 
+  def destroy
+    @membership_application.destroy
+    redirect_to membership_applications_url, notice: t('membership_applications.application_deleted')
```

Comment:
@thesuss - I *don't* trust google translate (GT) over you - it's just that it is the only way I can test English-to-Swedish translations.

Your judgement is the only one that counts :)